Dockerfile

```diff
@@ -1,5 +1,5 @@
 # docker buildx imagetools inspect --format='{{json .Manifest.Digest}}' node:current-alpine
-FROM node:22.0.0-alpine@sha256:9459e243f620fff19380c51497493e91db1454f5a30847efe5bc5e50920748d5
+FROM node:22.1.0-alpine@sha256:487dc5d5122d578e13f2231aa4ac0f63068becd921099c4c677c850df93bede8
```

Comment:
SHA doesn't match.

`docker buildx imagetools inspect --format='{{json .Manifest.Digest}}' node:current-alpine`  
`sha256:916b42f9e83466eb17d60a441a96f5cd57033bbfee6a80eae8e3249f34cf8dbe`

Reply:
`docker buildx imagetools inspect --format='{{json .Manifest.Digest}}' node:22.1.0-alpine`  
`sha256:487dc5d5122d578e13f2231aa4ac0f63068becd921099c4c677c850df93bede8`

Not sure why these are different hashes...

* [node:22.1.0-alpine](https://hub.docker.com/layers/library/node/22.1.0-alpine/images/sha256-321a52b9fccde74c532796f832046124af1dd997a65410d6155accd40fbe6ab8?context=explore)
* [node:current-alpine](https://hub.docker.com/layers/library/node/current-alpine/images/sha256-321a52b9fccde74c532796f832046124af1dd997a65410d6155accd40fbe6ab8?context=explore)

Reply:
Index digest difference is benign.